kalite/django_cherrypy_wsgiserver/cherrypyserver.py

```diff
@@ -142,7 +142,12 @@ def run_cherrypy_server(host="127.0.0.1", port=None, threads=None, daemonize=Fal
         # stop_server(pidfile)
 
         from django.utils.daemonize import become_daemon
-        become_daemon()
+        kalite_home = os.environ.get("KALITE_HOME", None)
+        logfile = os.path.join(kalite_home, "kalite.log") if (kalite_home and os.environ.get("NAIVE_LOGGING", False)) else None
```

Comment:
Unfortunately this implementation of `become_daemon` can only log to one file at once. I would also argue that if you want output to stdout, then you should not use `become_daemon` at all.